src/Tooltip/index.js

```diff
@@ -37,21 +37,21 @@ class Tooltip extends Component {
     if (this.state.match) {
       return (
         <>
-          <OnlyDesktop>
+          <OnlyDesktopInline>
             <DesktopView
               description={this.state.description}
               header={this.state.header}
               id={this.props.id}
               text={this.props.text}
             />
-          </OnlyDesktop>
-          <OnlyMobile>
```

Comment:
Are `OnlyDesktop` and `OnlyMobile` still in use somewhere else? Please remove them from src/styles.js otherwise.

Reply:
They are used a bunch of times in other places.